src/GoogleAuthenticator.php

```diff
@@ -86,12 +82,6 @@ public function checkCode($secret, $code): bool
         return $result > 0;
     }
 
-    /**
-     * NEXT_MAJOR: add the interface typehint to $time and remove deprecation.
```

Comment:
Your commit message says you removed phpdoc, yet you also remove this valuable piece of info, why?